xml/System.ServiceModel.Activities/IWorkflowInstanceManagement.xml

```diff
@@ -1092,10 +1092,10 @@
         <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
           <AttributeName>System.ServiceModel.OperationContract(Name="TransactedTerminate")</AttributeName>
         </Attribute>
-        <Attribute FrameworkAlternate="netframework-4.8">
+        <Attribute FrameworkAlternate="">
```

Comment:
what is this empty FrameworkAlternate @joelmartinez?

Reply:
.net framework 4.8 is still in early access (right?) … I'm **guessing** that something about this member changed … maybe the attribute was removed from 4.8?

Reply:
So should I just remove these empty entries @joelmartinez?

Reply:
@mairaw Yeah I _think_ it should be safe to remove it. What's odd is that mdoc [explicitly removes the attribute when it's all empty](https://github.com/mono/api-doc-tools/blob/43da2d7143f7a0d62019546b293ff2f82aee884e/mdoc/Mono.Documentation/MDocUpdater.cs#L3440-L3444)

```csharp
                        if (string.IsNullOrWhiteSpace (fxAttributeValue))
                        {
                            attr.ParentNode.RemoveChild (attr);
                            continue;
                        }
```

So this is a very interesting result indeed … it will be interesting to see what happens after the **next** CI run, after these are removed. In theory they shouldn't come back.